source/server.php

```diff
@@ -14,7 +14,7 @@
   $result_find_student = mysqli_query($db,$query_find_student);
   if (mysqli_num_rows($result_find_student) == 1) {
     $row = mysqli_fetch_assoc($result_find_student);
-    if($password == $row['password']){
+    if($password == md5($row['password'])){
```

Comment:
The passwords in DB must be stored in MD5 hash format. No need to find hash of hash(password in row from db). Also notice `$password` is already a hashed password, computed on line 12.